From 2e91d8c3c4bd34a27177180a38f62d3ba3c96031 Mon Sep 17 00:00:00 2001
From: Jeff Johnson <jjohnson@qti.qualcomm.com>
Date: Fri, 23 May 2025 09:41:02 -0700
Subject: [PATCH] ath12k: WCN7850 hw2.0: update to
 WLAN.HMT.1.1.c5-00284.1-QCAHMTSWPL_V1.0_V2.0_SILICONZ-3

Fix firmware load fail issue with some versions of WCN7850.

Closes: https://bugzilla.kernel.org/show_bug.cgi?id=220108
Closes: https://bugzilla.redhat.com/show_bug.cgi?id=2365882
Closes: https://www.reddit.com/r/archlinux/comments/1kja6f9/ath12k_regression_on_latest_linuxfirmware_upgrade/
Reported-by: Walt Holman <waltholman09@gmail.com>
Closes: https://lore.kernel.org/all/CAMb39_m5Wm-5xjxYEJg4RxPZc1rdq0BkAJk_Q0B9APC3CAeORA@mail.gmail.com/
Reported-by: Benjamin Eggerstedt <benjamin.eggerstedt@gmail.com>
Closes: https://lore.kernel.org/all/EBAC1B61-E726-4614-9709-268CFF07007F@gmail.com/
Fixes: aea9e010506e ("WCN7850 hw2.0: 1.1: add WLAN.HMT.1.1.c5-00284-QCAHMTSWPL_V1.0_V2.0_SILICONZ-3")
Signed-off-by: Jeff Johnson <jjohnson@qti.qualcomm.com>
---
 WHENCE                        |   2 +-
 ath12k/WCN7850/hw2.0/amss.bin | Bin 6082624 -> 6078528 bytes
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/WHENCE b/WHENCE
index 865552e9..93f849c4 100644
--- a/WHENCE
+++ b/WHENCE
@@ -3963,7 +3963,7 @@ Driver: ath12k - Qualcomm Technologies Wi-Fi 7 chipset support
 File: ath12k/WCN7850/hw2.0/board-2.bin
 File: ath12k/WCN7850/hw2.0/amss.bin
 File: ath12k/WCN7850/hw2.0/m3.bin
-Version: WLAN.HMT.1.1.c5-00284-QCAHMTSWPL_V1.0_V2.0_SILICONZ-3
+Version: WLAN.HMT.1.1.c5-00284.1-QCAHMTSWPL_V1.0_V2.0_SILICONZ-3
 File: ath12k/WCN7850/hw2.0/Notice.txt
 File: ath12k/QCN9274/hw2.0/board-2.bin
 File: ath12k/QCN9274/hw2.0/firmware-2.bin
